File: points_of_interest/v1/get/pois/iOS SDK/PointsOfInterest.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// What are the popular places in Barcelona?
amadeus.referenceData.locations.pointsOfInterest.get(params: ["latitude": "41.397158",
    "longitude": "2.160873",
    "radius": "2"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })

File: points_of_interest/v1/get/pois/Node SDK/points_of_interest.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What are the popular places in Barcelona (based on a geo location and a radius)
    const response = await amadeus.referenceData.locations.pointsOfInterest.get(
      {
        latitude: 41.397158,
        longitude: 2.160873,
      }
    );

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
